workflows/computational-chemistry/protein-ligand-complex-parameterization/protein-ligand-complex-parameterization.ga

```diff
@@ -77,6 +76,253 @@
             "uuid": "277627e0-eba8-4de5-855b-63711c0db691",
             "workflow_outputs": []
         },
+        "10": {
```

Comment:
@simonbray This does look suspiciously like the order is messed up because the sorting wasn't numeric ... any chance you could see if that could be fixed ? That might be more feasible in the short term than switching to gxformat2

Reply:
Yes, thanks for spotting that!!